pages/web-literacy/color.js

```diff
@@ -0,0 +1,33 @@
+var categories = require('./categories');
```

Comment:
Do we still rely on this? I don't think we do anymore given that we're now using CSS for the coloring. Can we remove this? (and do we need to remove mention of a colors object for the react-circletree package?)